RecoTauTag/Configuration/python/updateHPSPFTaus.py

```diff
@@ -0,0 +1,53 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
the name of this file requires some disambiguation.
There is also updateHPSPFTaus_cff with completely different content.

perhaps customiseHPSPFTausForPAT  ?

Reply:
I've followed your proposals. 
Cheers,
Roger